code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -356,7 +356,7 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 	return total_transfered
 */
 
-/datum/reagents/proc/metabolize(var/mob/M, var/alien)
+/datum/reagents/proc/metabolize(var/mob/living/M, var/alien)
```

Comment:
You really want to talk to the rest of the team before changing a signature like this.

Reply:
It's called on a living mob in every single case, so it should be fine

Reply:
There goes the spooky ghost bomb bus.

Reply:
This should have been `/mob/living` from the start.